plugin/trino-hive-hadoop2/pom.xml

```diff
@@ -91,6 +91,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino.tpch</groupId>
+            <artifactId>tpch</artifactId>
+            <scope>test</scope>
```

Comment:
i missed that earlier, but  why actually not add this test in `trino-hive` module, where most of the tests live?

The existing test next to yours (`TestCachingHiveMetastoreWithQueryRunner`) that lives in `trino-hive-hadoop2` is there probably by mistake. Note that its name is non-standard, so it could be miss during review.